app/views/staff/reports_state/review/complete.html.haml

```diff
@@ -1,17 +1,22 @@
-= content_for :page_title_prefix, t("page_title.report.in_review.complete", report_description: @report_presenter.description, report_financial_quarter: @report_presenter.financial_quarter_and_year)
+= content_for :page_title_prefix, t("page_title.report.in_review.complete", report_organisation: @report.organisation.name, report_financial_quarter: @report_presenter.financial_quarter_and_year)
 
 %main.govuk-main-wrapper#main-content{ role: "main" }
   .govuk-grid-row
     .govuk-grid-column-two-thirds
       .govuk-panel.govuk-panel--confirmation
         %h1.govuk-panel__title
-          = t("action.report.in_review.complete.title")
-
-        .govuk-panel__body
-          = t("action.report.activate.complete.body", report_description: @report_presenter.description, report_financial_quarter: @report_presenter.financial_quarter_and_year)
+          = t("action.report.in_review.complete.title", report_financial_quarter: @report_presenter.financial_quarter_and_year, report_organisation: @report.organisation.name)
 
       %h2.govuk-heading-m
         What happens next
 
       %p.govuk-body
-        This report is now in review
+        The report is now in review with BEIS.
+        %br
```

Comment:
I would have just made this a new paragraph?

Reply:
I had the same thought, but I am following the way the client gave me the text, which contains a line break but not a new paragraph 🤷‍♀️